integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/MonitoringUtils.java

```diff
@@ -306,19 +306,46 @@ public static void editPrometheusCM(String oldRegex, String newRegex,
    *                            default is regex: default;domain1
    * @return the prometheus Helm installation parameters
    */
+  public static PrometheusParams installAndVerifyPrometheus(String promReleaseSuffix,
+                                                            String promNamespace,
+                                                            String promVersion,
+                                                            String prometheusRegexValue,
+                                                            String promHelmValuesFile) {
+    return installAndVerifyPrometheus(promReleaseSuffix,
+            promNamespace,
+            promVersion,
+            prometheusRegexValue,
+            promHelmValuesFile,
+            null);
+  }
+
+  /**
+   * Install Prometheus and wait up to five minutes until the prometheus pods are ready.
+   *
+   * @param promReleaseSuffix the prometheus release name unigue suffix
+   * @param promNamespace the prometheus namespace in which the operator will be installed
```

Comment:
The comment is confusing. Is promNamespace for Promethus or operator?

Reply:
fixed